certified-connectors/DocuSignDemo/script.csx

```diff
@@ -4921,7 +4921,7 @@ private void RenameSpecificKeys(JObject jObject, Dictionary<string, string> keyM
     {
       foreach (var field in body)
       {
-        if (field["fieldType"].ToString() == "Table row")
+        if ((field["fieldType"] != null) && (field["fieldType"].ToString() == "Table row"))
```

Comment:
Do we want to ignore case or maybe the space as well when comparing?

Reply:
Not needed. The swagger has enum declared with these values. There is no way for the user to enter a text. 
`"fieldType": {
                    "type": "string",
                    "x-ms-summary": "Type",
                    "description": "Type of the Docgen field",
                    "enum": [
                      "Text box",
                      "Date",
                      "Number",
                      "Select",
                      "Table row"
                    ]
                  },`

Reply:
sounds good. Wanted to double check.